server-common/src/main/java/org/apache/kafka/server/common/EligibleLeaderReplicasVersion.java

```diff
@@ -25,7 +25,7 @@ public enum EligibleLeaderReplicasVersion implements FeatureVersion {
     ELRV_0(0, MetadataVersion.MINIMUM_KRAFT_VERSION, Collections.emptyMap()),
 
     // Version 1 enables the ELR (KIP-966).
-    ELRV_1(1, MetadataVersion.IBP_4_0_IV1, Collections.emptyMap());
+    ELRV_1(1, MetadataVersion.IBP_4_1_IV0, Collections.emptyMap());
```

Comment:
I think the comment should be something like "enables ELR by default for new clusters"